### PR TITLE
front: editor: add direction, remove angles, for PSL Signs

### LINKED ISSUE #6600: PSL edition fails because of usage of deprecated angle fields
### What happened?

When trying to create or edit a PSL, it doesn't work. 

### What did you expect to happen?

_No response_

### How can we reproduce it (as minimally and precisely as possible)?

1. Go to the editor
2. Create a PSL
3. Try to save (it doesn't work because of unexpected field `angle_geo` and `angle_sch`

### What operating system, browser and environment are you using?

- **Browser**: Firefox v109
- **OS**: Nixos v22.11
- **Env**: Local & Dev & Staging

### OSRD version (top right corner `Account` button > `Informations`)

086240a

### PR DESCRIPTION
**Pull request by nicolaswurtz**
Closes #6600 

- Angles are not geo/sch anymore.
- Angles are not needed anymore in points editions.
- Direction is needed in PSL Sign card

### CODE REVIEW

File: front/public/locales/fr/infraEditor.json
Comment (clarani) [inline front/public/locales/fr/infraEditor.json:1]: This file should be generated by the backend, is it what you did ?

File: front/src/applications/editor/tools/rangeEdition/speedSection/PslSignCard.tsx [hunk 1]
```diff
@@ -97,6 +69,17 @@ const PslSignCard = ({
             sm
           />
         </div>
+        <div className="my-2">
+          <SelectImprovedSNCF
+            label={t('Editor.tools.speed-edition.sign-direction').toString()}
```
Comment (SharglutDev) [inline front/src/applications/editor/tools/rangeEdition/speedSection/PslSignCard.tsx:74]: It seems there is no trad for this

Comment (nicolaswurtz) [reply front/src/applications/editor/tools/rangeEdition/speedSection/PslSignCard.tsx:74]: done!

File: front/public/locales/en/translation.json [hunk 2]
```diff
@@ -415,8 +415,8 @@
         "main-speed-limit": "Default speed limit",
         "move-range-extremity": "Click to move the end:",
         "new-tag": "Composition code",
-        "sign-angle-geo": "Angle Geo",
-        "sign-angle-sch": "Angle Sch",
+        "sign-angle": "Angle",
```
Comment (clarani) [inline front/public/locales/en/translation.json]: You don't need this traduction
```suggestion
```

File: front/public/locales/fr/translation.json [hunk 3]
```diff
@@ -413,8 +413,8 @@
         "main-speed-limit": "Limitation de vitesse par défaut",
         "move-range-extremity": "Cliquer pour déplacer l'extrémité :",
         "new-tag": "Code de composition",
-        "sign-angle-geo": "Angle Geo",
-        "sign-angle-sch": "Angle Sch",
+        "sign-angle": "Angle",
```
Comment (clarani) [inline front/public/locales/fr/translation.json]: You don't need this translation
```suggestion
```

File: front/src/applications/editor/tools/pointEdition/tools.ts
Comment (clarani) [inline front/src/applications/editor/tools/pointEdition/tools.ts:1]: You can also remove the prop requiresAngle l34

File: front/src/applications/editor/tools/rangeEdition/speedSection/EditPSLSection.tsx [hunk 4]
```diff
@@ -29,7 +29,6 @@ const getNewAnnouncementSign = (
   const speedInKmH = msToKmh(speedLimit);
   const speedMultipleOfFive = Math.ceil(speedInKmH / 5) * 5;
   return {
-    angle: 0,
```
Comment (clarani) [inline front/src/applications/editor/tools/rangeEdition/speedSection/EditPSLSection.tsx:32]: Can you add a direction by default ? Here and l47 please

File: front/src/applications/editor/tools/pointEdition/tools.ts
Comment (nicolaswurtz) [reply front/src/applications/editor/tools/pointEdition/tools.ts:1]: already done!

File: front/src/applications/editor/tools/rangeEdition/speedSection/EditPSLSection.tsx (hunk 4 above)
Comment (nicolaswurtz) [reply front/src/applications/editor/tools/rangeEdition/speedSection/EditPSLSection.tsx:32]: done!

File: front/public/locales/fr/infraEditor.json
Comment (nicolaswurtz) [reply front/public/locales/fr/infraEditor.json:1]: I don't think so?

Comment (nicolaswurtz) [reply front/public/locales/fr/infraEditor.json:1]: No no, this file is manually edited (and it have to!).

**Review by SharglutDev — APPROVED** [review]
Lgtm and tested

File: front/public/locales/fr/infraEditor.json
Comment (clarani) [reply front/public/locales/fr/infraEditor.json:1]: My bad, thanks !

**Review by clarani — APPROVED** [review]
LGTM

**Pull request merged**